### DISCUSSION TITLE
multiple devices

### DISCUSSION THREAD

**Discussion by hoefty78** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#564] [General]
Hi at all.
is it possible to choose more than one device.

I have 2 Xiaomi Vacuum (gen 1) in my house. One every floor.

I can´t find anything.

Thanx

**Comment by PiotrMachowski** [comment]
https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor/issues/508
